_pages/team.md

```diff
@@ -46,13 +46,13 @@ He is multidisciplinary trained in Pre-clinical Medicine, Neural Engineering, Me
 
-Dr Srija is a Clinical Pharmacist (Doctor of Pharmacy) by profession, trained in Biz ops like Regulatory Compliance, Banking Operations, HR management, Public Relations, Business development Collaterals design, Awareness and Marketing Campaign design and other technical aspects like Gitops, Website building etc.. She is skilled in Systemic and Conversational Peer Review and has been a Reviewer for a few international conferences and journals including International Society of Pharmacoeconomics and Outcomes Research 6 TH Latin America Conference, Brazil - 2017 and 20th Annual European Congress.-2017.
+Dr Srija is a Clinical Pharmacist (Doctor of Pharmacy) by profession, trained in biz-ops like Regulatory Compliance, Banking Operations, HR Management, Public Relations, Business Development, Collaterals design, Awareness and Marketing Campaign Design and other technical aspects like GitOps, Website development among others, She is skilled in Systemic and Conversational Peer Review and has been a Reviewer for a few international conferences and journals including International Society of Pharmacoeconomics and Outcomes Research - 6th Latin America Conference, Brazil - 2017 and 20th Annual European Congress - 2017.
 
-Her professional Interest includes Artificial Intelligence and Insilico based accelerated drug discovery, Medical devices, Digihealth, Telemedicine, IoMT. Research interests include Psychoneuroimmunological studies with respect to human epigenetics and social habits. She aspires for Social Welfare and has conducted various Healthcare Awareness Campaign during her graduation days. She actively coordinates the activities of TWISHA - The Women in Self Health Awareness, a health4all initiative by an NGO, Gramsheel Foundation. She has been one of among the top distinction holder of the prestigious Ashoka Changemaker Intraprenuership - 2018
+Her professional Interest includes Artificial Intelligence and Insilico Drug Discovery, Medical Devices, Digihealth, Telemedicine, IoMT. Research interests include Psychoneuroimmunological studies with respect to human epigenetics and social habits. She aspires for Social Welfare and has conducted various Healthcare Awareness Campaign during her graduation days. She actively coordinates the activities of TWISHA - The Women in Self Health Awareness, a health4all initiative by an NGO, Gramsheel Foundation. She has been one among the top distinction holders of the prestigious Ashoka Changemaker Intraprenuership - 2018
```

Comment:
Interest is still capitalized.
Interest includes -> interests include (grammar) change not done